test/e2e/containers_conf_test.go

```diff
@@ -788,7 +788,7 @@ var _ = Describe("Verify podman containers.conf usage", func() {
 	})
 
 	startContainer := func(params ...string) string {
-		args := []string{"create"}
+		args := []string{"run"}
```

Comment:
ah sorry you will need to add "-d" as arg as well so it still prints the container id as output.